assets/js/googlesitekit/modules/datastore/sharing-settings.test.js

```diff
@@ -282,6 +282,25 @@ describe( 'core/modules sharing-settings', () => {
 				);
 			} );
 		} );
+
+		describe( 'rollbackSharingSettings', () => {
+			it( 'dispatches a request to rollback the sharing settings to the currently saved values', () => {
+				registry
+					.dispatch( CORE_MODULES )
+					.receiveGetSharingSettings( sharingSettings );
+
+				// Make changes to the shared settings to verify the rollback.
+				registry
+					.dispatch( CORE_MODULES )
+					.setSharedRoles( 'analytics', [ 'editor', 'subscriber' ] );
+
```

Comment:
It would be good to add an assertion here that the `sharingSettings != savedSharingSettings` – essentially ensuring the accuracy of the comment above.

Reply:
Done, thank you!